### ISSUE TITLE
lottie: memory leak reported by XCode Instruments

### ISSUE THREAD

**Issue by RuiwenTang** [thorvg/thorvg#3070]
A possible memory leak was found by XCode instruments, can help confirm it?
<img width="1705" alt="截屏2024-12-23 上午10 04 00" src="https://github.com/user-attachments/assets/ac26d5b7-3023-46ed-b305-d35963f11867" />

**Comment by hermet** [comment]
@RuiwenTang Hi, tested with `Lottie` in the example. right?

**Comment by RuiwenTang** [comment]
> @RuiwenTang Hi, tested with `Lottie` in the example. right?

Yes, or just test with the guitar.json or expressions/intelia_logo_animation.json
